src/main/java/frc/robot/subsystems/PizzaBox/PizzaBoxSubsystem.java

```diff
@@ -65,12 +64,12 @@ public Command stopKicker() {
 
   public Command speedyArm_Command(DoubleSupplier f) {
     if (f.getAsDouble() > 99 && f.getAsDouble() < 261) {
-      return this.runOnce(() -> pizzaBoxIO.setMotor(1))
+      return this.runOnce(() -> pizzaBoxIO.setMotor(80))
```

Comment:
setMotor() parameter unit is in percentage -1 to 1; should not be 80.

Also this command is not working as intended; it does not spin the pizzabox while arm is swinging